package.json

```diff
@@ -112,21 +112,21 @@
     "express-logging": "1.1.1",
     "extract-zip": "2.0.1",
     "fastest-levenshtein": "1.0.16",
-    "fastify": "4.28.1",
+    "fastify": "4.29.0",
     "find-up": "7.0.0",
     "flush-write-stream": "2.0.0",
     "folder-walker": "3.2.0",
     "from2-array": "0.0.4",
     "fuzzy": "0.1.3",
-    "get-port": "5.1.1",
+    "get-port": "7.1.0",
     "gh-release-fetch": "4.0.3",
     "git-repo-info": "2.1.1",
     "gitconfiglocal": "2.1.0",
     "http-proxy": "1.18.1",
     "http-proxy-middleware": "2.0.7",
     "https-proxy-agent": "7.0.6",
-    "inquirer": "6.5.2",
-    "inquirer-autocomplete-prompt": "1.4.0",
+    "inquirer": "9.1.0",
+    "inquirer-autocomplete-prompt": "3.0.1",
```

Comment:
changed to esm and supports inquirer >= 9: https://github.com/mokkabonna/inquirer-autocomplete-prompt/releases

Reply:
ugh, after going down a rat hole I ended backing out of the inquirer-autocomplete-prompt upgrade since I couldn't figure what was changing and why... which in turn forced me to downscale the inquirer upgrade to v8 😞